net-misc/autossh required *FLAGS-=-flto*

net-misc/autossh compilation fails with `/usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-misc/autossh-1.4g/temp/autossh.uWYPkV.ltrans0.ltrans.o: in function 'ssh_watch':
<artificial>:(.text+0xfbf): undefined reference to 'setproctitle'` unless `*FLAGS-=-flto*` is used to override.

--- a/sys-config/ltoize/files/package.cflags/lto.conf
+++ b/sys-config/ltoize/files/package.cflags/lto.conf
@@ -58,6 +58,7 @@ media-video/ffmpeg *FLAGS-=-flto* #NOTE: Depending on your CPUFLAGS this may wor
 media-video/mplayer *FLAGS-=-flto*
 net-libs/webkit-gtk:3 *FLAGS-=-flto*
 net-libs/webkit-gtk:4 *FLAGS-=-flto*
+net-misc/autossh *FLAGS-=-flto* # Undefined references
 net-misc/dhcp "has server ${IUSE//+} && use server && FlagSubAllFlags -flto*" # Cannot be built with LTO by default, but can if "server" USE is disabled
 net-misc/lksctp-tools *FLAGS-=-flto* # function `main': <artificial>:(.text.startup+0x16de): undefined reference to `sctp_connectx'
 net-misc/nextcloud-client *FLAGS-="-flto"* # Segfaults at start with libQt5Core.so

